pkg/cmd/pr/create/regex_writer.go

```diff
@@ -0,0 +1,62 @@
+package create
+
+import (
+	"bufio"
+	"bytes"
+	"io"
+	"regexp"
+)
+
+func NewRegexWriter(out io.Writer, regexp *regexp.Regexp, repl string) io.Writer {
```

Comment:
the `regexp` input argument shadows the standard library package, it's common to use something like `re` instead so it's easier to tell what's being referred to when just glancing at the code